scripts/profile_scraper/Sorter.py

```diff
@@ -0,0 +1,39 @@
+import os
+import shutil
+import datetime
+
+path = "/Users/kris/Documents/CSSA/cssa-web/cssa.aei/" # path for the folder containing all the files, unsorted
```

Comment:
This path would only work on your machine right, so we'd have to maybe turn this into an Environment Variable? Will come back to it